src/components/Modals/AddContactModal.jsx

```diff
@@ -12,7 +11,6 @@ import FormControl from '@mui/material/FormControl';
 import IconButton from '@mui/material/IconButton';
 import InputAdornment from '@mui/material/InputAdornment';
 import TextField from '@mui/material/TextField';
-import { runNotification } from '@utils';
 import { FormSection } from '../Form';
```

Comment:
Can we split these imports with comments, just to keep all files consistent?  Got component and hook in Material UI